includes/assets.php

```diff
@@ -243,6 +243,28 @@ public function register_scripts() {
 				);
 			}
 
+			wp_register_script(
+				'scf-commands-admin',
+				acf_get_url( 'assets/build/js/commands/scf-admin' . $suffix . '.js' ),
+				array( 'acf', 'wp-plugins', 'wp-element', 'wp-components', 'wp-data', 'wp-commands', 'wp-i18n', 'wp-dom-ready' ),
```

Comment:
why do these scripts depend on the "act" script?

Reply:
I guess because they need the acf object to be there, as its being extended with the localized data.

Reply:
To maintain the proper script loading order and ensure things like `window.acf.data` already exist.

Reply:
Yep, if we relied on an endpoint we wouldn't need this. The only "real" dependency on ACF is to get the CPTs created by SCF from the DB.